src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,15 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
+    {comments.length !== 0
+      ? comments.map(comment => (
```

Comment:
Move this check to the PostInfo component and render CommentList or "no comments" message there